CMakeLists.txt

```diff
@@ -106,7 +106,9 @@ string(REGEX REPLACE     "${protobuf_VERSION_REGEX}" "\\3"
 string(REGEX REPLACE     "${protobuf_VERSION_REGEX}" "\\5"
   protobuf_VERSION_PRERELEASE "${protobuf_VERSION_STRING}")
 
-message(STATUS "${protobuf_VERSION_PRERELEASE}")
+if(protobuf_VERSION_PRERELEASE)
```

Comment:
Could we just move this to the existing if(protobuf_VERSION_PRERELEASE) block on line 17 instead?

Reply:
Sure, let me create a new PR as I think the codebase has moved on quite a bit since I created this one.